include/navigation/insilico_rotater.h

```diff
@@ -91,7 +90,7 @@ struct InSilicoRotater
         const size_t m_ScanStep;
         const IterType m_BeginRoll, m_EndRoll;
         const cv::Mat &m_Image, &m_MaskImage;
-        static OMP::Array<cv::Mat> m_ScratchImages, m_ScratchMaskImages;
+        static cv::Mat m_ScratchImages, m_ScratchMaskImages;
```

Comment:
how come there's function scope and class scope statics here? As far as I can tell these aren't actually being initialised anywhere...

Reply:
Oops, forgot to take these out